modules/social_features/social_group/modules/social_group_request/social_group_request.module

```diff
@@ -299,29 +300,32 @@ function social_group_request_preprocess_activity(array &$variables) {
   $activity = $variables['elements']['#activity'];
 
   if (!$activity->get('field_activity_message')->isEmpty()) {
+    $message_storage = \Drupal::entityTypeManager()->getStorage('message');
     /** @var Drupal\message\Entity\Message $message */
-    $message = $activity->field_activity_message->entity;
+    $message = $message_storage->load($activity->field_activity_message->target_id);
```

Comment:
@DmytroRovovyi I'd recommend to use `$activity->get('field_activity_message')->getValue()` for getting value from field.

Reply:
@Ressinel changed field_activity_message to get value from field.